isodesign/io.py

```diff
@@ -20,8 +20,54 @@
 
 """
 
+import pandas as pd 
+from pathlib import Path
 
+class IoGestion:
+    def __init__(self):
+        self.netw = None
+        self.tvar = None
+        self.mflux = None
 
+    def read_file(self, data : str):
+        if isinstance(data, str):
+            data_path = Path(data).resolve()
+            if data_path.exists(): 
+                if data_path.suffix == ".netw":
+                    self.netw = pd.read_csv(data, sep='\t', skiprows=[0], header=None)
+                if data_path.suffix == ".tvar":
+                    self.tvar = pd.read_csv(data, sep='\t')
+                if data_path.suffix == ".mflux":
+                    self.mflux = pd.read_csv(data, sep='\t', skiprows=[0])
+                else:
+                    raise TypeError (f"{data_path} is not in the good format\n Only .netw, .tvar, .mflux formats are accepted")
```

Comment:
Attention, ici ton else ne va que agir sur le dernier if! Si tu veux checker le format du fichier, tu peux commencer plus haut en notant:
`if data_path.suffix not in [".netw", ".tvar", ".mflux"]:`
` 
raise TypeError (f"{data_path} is not in the good format\n Only .netw, .tvar, .mflux formats are accepted")
`